### DISCUSSION TITLE
feat/docus-docs standalone

### DISCUSSION THREAD

**Discussion by oripka** [nuxt/nuxt#16856] [Questions]
I try to get the docs in `feat/docus-docs` which already use Nuxt3 to work standalone. So basically just copying the `/docs` folder and then running the project with Nuxt3 as a dependency.

I am having trouble setting up the environment using commands like `yarn add`, `yarn install`, and `yarn run dev`. Getting errors like `Cannot find any nuxt version from...`. I tried to do `yarn add` for nuxt and nuxt-kit manually resulting in more errors.

**Comment by danielroe** [comment]
See https://v3.nuxtjs.org/community/framework-contribution#set-up-your-local-development-environment.
